app/gitcoinbot/views.py

```diff
@@ -56,7 +56,7 @@ def payload(request):
         comment_text = comment_dict.get('body')
         issue_id = request_json.get('issue', {}).get('number')
         installation_id = request_json.get('installation', {}).get('id')
-        # sender = request_json['sender']['login']
+        sender = request_json['sender']['login']
```

Comment:
Can you switch this to something like `sender = request_json.get('sender', {}).get('login', '')`